subprojects/core/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -694,7 +694,7 @@ public boolean isIgnoreExitValue() {
      */
     @Internal
     @Incubating
-    public Provider<ExecResult> getExecResult() {
+    public Provider<ExecResult> getExecutionResult() {
```

Comment:
Can't we just remove this method?

Reply:
No, JavaExec and Exec don't share AbstractExecTask, so they both need it.